lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js

```diff
@@ -29,8 +29,10 @@ class ExternalAnchorsUseRelNoopenerAudit extends Audit {
     return {
       category: 'Performance',
       name: 'external-anchors-use-rel-noopener',
-      description: 'Site opens external anchors using rel="noopener".',
-      helpText: 'Anchors that open in a new tab should use <code>target="_blank"</code> and <code>rel="noopener"</code> so the <a href="https://jakearchibald.com/2016/performance-benefits-of-rel-noopener" target="_blank">opening page\'s performance does not suffer.</a>',
+      description: 'Site opens external anchors using rel="noopener"',
+      helpText: 'Open new tabs using <code>rel="noopener"</code> to improve performance and ' +
```

Comment:
Yea, this is the problem with template strings. It'll be inclusive of whitespace. You could break up links and concat those parts`+ ... +`, but I'm not sure it's worth the trouble to mix and match template strings and str concat.